File: go/wam/6242.go
const WamPsGroupSafetyCheckUiInteractions = 6242 // channel: private

type PsGroupSafetyCheckUiInteractionsEvent struct {
	PsSafetyCheckGroupJid string `wam:"ps_safety_check_group_jid"` // field 1
	PsSafetyCheckInteraction PSGROUPSAFETYCHECKUIINTERACTIONS `wam:"ps_safety_check_interaction"` // field 2
	IntegrityGroupUserHashedId string `wam:"integrity_group_user_hashed_id"` // field 3
}

File: ts/wam/6242.ts
export const WamPsGroupSafetyCheckUiInteractions = 6242 as const

export interface PsGroupSafetyCheckUiInteractionsEvent {
  /** field 1, wire `ps_safety_check_group_jid` */
  psSafetyCheckGroupJid?: string
  /** field 2, wire `ps_safety_check_interaction` */
  psSafetyCheckInteraction?: PSGROUPSAFETYCHECKUIINTERACTIONS
  /** field 3, wire `integrity_group_user_hashed_id` */
  integrityGroupUserHashedId?: string
}